common/include/pcl/point_cloud.h

```diff
@@ -703,26 +695,10 @@ namespace pcl
       inline Ptr
       makeShared () const { return Ptr (new PointCloud<PointT> (*this)); }
 
-    protected:
-      /** \brief This is motivated by ROS integration. Users should not need to access mapping_.
-        * \todo Once mapping_ is removed, erase the explicitly defined copy constructor in PointCloud.
```

Comment:
I think I overlooked this.

Reply:
Yup. Please remove also 
https://github.com/PointCloudLibrary/pcl/blob/29c6f61454c6a279c0df8c3efd1334be19446293/common/include/pcl/point_cloud.h#L181-L193